src/core/lv_obj_pos.c

```diff
@@ -319,13 +319,26 @@ void lv_obj_align(lv_obj_t * obj, lv_align_t align, int32_t x_ofs, int32_t y_ofs
     lv_obj_set_pos(obj, x_ofs, y_ofs);
 }
 
+static void add_anchored_children(lv_obj_t * obj, const lv_obj_t * base)
```

Comment:
Please rename to `add_anchored_child` and move to the section in the file called `STATIC FUNCTIONS` and add the function prototype to `STATIC PROTOTYPES`